src/components/Apigw/CFiltersDropdown.vue

```diff
@@ -48,10 +48,12 @@ export default {
   },
   methods: {
     onAddFilter (func) {
-      const add = { ...func, params: [] }
+      const add = { ...func, created: true, params: [] }
       const { params = [] } = func
-      for (const p of params) {
-        add.params.push({ ...p, options: { ...p.options } })
+      if (func.params) {
```

Comment:
no need for this if since params will always be [] even if it doesn't exist that's why the line 52 is there